src/dmd/target.d

```diff
@@ -571,6 +573,157 @@ struct Target
     {
         return .toArgTypes(t);
     }
+
+    /**
+     * Returns:
+     *   true if return value from function is on the stack
+     */
+    extern (C++) static bool isReturnOnStack(TypeFunction tf)
```

Comment:
Although it's pretty obvious, you usually complain of missing `Params:` section 😉.